pkg/web/apps/apps_service.go

```diff
@@ -34,3 +35,16 @@ func (a *appsService) GetApps(c echo.Context) (*[]models.App, error) {
 
 	return apps, nil
 }
+
+// GetAppsId retrieves app by id from the repository
+func (a *appsService) GetAppsID(id string) (*[]models.App, string) {
```

Comment:
Same as the previous comment. Should be renamed to GetAppbyId in my opinion.

Reply:
:+1: